### DISCUSSION TITLE
Game Crash on installation on mac

### DISCUSSION THREAD

**Discussion by TejedorWeb** [Monika-After-Story/MonikaModDev#9927] [QA - tech support + general]
Each time I try to install the game as stated in faq and instructions the game will drop this message
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/0utils.rpy", line 40, in <module>
ImportError: cannot import name handlers

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/script.py", line 753, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=fn + source)
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/script.py", line 445, in finish_load
    node.early_execute()
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/ast.py", line 865, in early_execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/DDLC.app/Contents/Resources/autorun/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/0utils.rpy", line 40, in <module>
ImportError: cannot import name handlers

Darwin-17.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.14

**Comment by Booplicate** [comment]
Seems like you installed it incorrectly, try install again using our installer.

**Reply by TejedorWeb** [reply]
Didn't use the installer, Terminal does not detect it, tried using the Zip and this happened

**Reply by Booplicate** [reply]
I figured you didn't use it. What do you mean by "Terminal does not detect it"? It shouldn't detect anything.

**Reply by TejedorWeb** [reply]
the installer will just display a .txt of data when trying to run it, when trying to use Terminal to run it using the ./ command, it won't detect it.

**Reply by Booplicate** [reply]
> the installer will just display a .txt of data when trying to run it

That means you opened it for reading, not ran.

> when trying to use Terminal to run it using the ./ command, it won't detect it

You need to specify the full path to the file to execute it, if you're in its directory, you can just write its name `./mas-installer-mac`. Read more about it [here](https://github.com/Monika-After-Story/mas-installer/wiki/Launch-Instructions).

**Reply by TejedorWeb** [reply]
thanks, it seems my system is too old to run it.

Abort trap: 6
MacBook-Air-de-Jose-3:Downloads JoseArmando$ ./mas-installer-mac
dyld: lazy symbol binding failed: Symbol not found: _objc_alloc_init
  Referenced from: /Users/JoseArmando/Downloads/./mas-installer-mac (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib

dyld: Symbol not found: _objc_alloc_init
  Referenced from: /Users/JoseArmando/Downloads/./mas-installer-mac (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libobjc.A.dylib

**Reply by Booplicate** [reply]
Interesting, somehow you're missing a shared library.

Then try to install manually, you need to merge the mod files with the DDLC files. You might need to hold the options key to be able to merge.

**Reply by TejedorWeb** [reply]
All right, do I need to merge it with the “autorun” carpet or the “game” one?

**Reply by Booplicate** [reply]
You need to put the mod files into `DDLC.app/Contents/Resources/autorun/` and merge 2 `game/` folders.

**Reply by TejedorWeb** [reply]
it worked, how will I update the game when needed without the file though?

**Reply by Booplicate** [reply]
Use the in-game updater, it will do the work for you.
